### DISCUSSION TITLE
Enterprise Services Idea

### DISCUSSION THREAD

**Discussion by dpaez** [geut/chan#71] [Ideas]
Hello devs :wave: 

**We need your help!** We'd like to know more about how are you using `chan`. This can help us offer a more tailored solution for companies. We ([GEUT](https://geutstudio.com)) are primarily devs and we started building tools for devs, like chan, [xd](https://github.com/geut/xd), and [many others](https://github.com/geut/). 

We are thinking to bring a major update to some of these modules and have some ideas for integrations, like glue modules to make _DX_ way better. We are also thinking on creating a set of tools to work around the `release` lifecycle of node.js projects. 

Below there are some questions, feel free to answer these or add your own comments. **Thanks a lot!** ⭐ 🙏 

PS: all of these modules will remain open source. We are just thinking of adding some integrations modules and offer an `enterprise` paid version that adds some value for this kind of customers. Sounds interesting? Let us know!

**About Chan Usage**
- Do you use `chan` at work or just for personal projects? 
- Are you using it with other languages (outside of JS)?
- Is releasing software a pain point for you (if it is, what's the worst part, add some comments)?
- Do you share your release notes with every release?
- Are you releasing against self-hosted infra (ie: digital-ocean-alike, custom hard/soft) or using cloud services (aws, vercel, netlify)?
- If interested in enterprise support, would you help us connect with your company management?

**Comment by dpaez** [comment]
Pinging @JacobDB, @michaeldbianchi maybe this could be interesting for you...

**Comment by JacobDB** [comment]
We're a very small company, so I don't think we'd be interested in paying for a service, but here's my survey response:

About Chan Usage

- **Do you use chan at work or just for personal projects?** Both for work and personal projects
- **Are you using it with other languages (outside of JS)?** I use it for any project I'm working on, primarily custom WordPress themes (built with a combination of JS, CSS, and PHP)
- **Is releasing software a pain point for you (if it is, what's the worst part, add some comments)?** Generating a release package isn't too bad, but deployment in particular is a pain because we still use FTP for everything, and as such automated deployment tools aren't really viable with our systems.
- **Do you share your release notes with every release?** We don't release code publicly, so not really, the  closest to this would just be updating the CHANGELOG with each release.
- **Are you releasing against self-hosted infra (ie: digital-ocean-alike, custom hard/soft) or using cloud services (aws, vercel, netlify)?** Self-hosted LAMP servers, uploading via FTP.
- **If interested in enterprise support, would you help us connect with your company management?** We're just too small of a company to necessitate enterprise-level support, but depending on the services provided, I'd consider it.

**Comment by dpaez** [comment]
Cool @JacobDB. Thank you very much for your response! 🙌

**Comment by lud** [comment]
* **Do you use chan at work or just for personal projects?**

  I use it mainly on personal project but I recently introduced `chan` to my
  team, as we are now maintaining a global changelog for several microservices
  apps.

* **Are you using it with other languages (outside of JS)?**

  I never use it for JS actually, as I rarely code in JS. Our main languages are
  currently Java and Elixir. We use the CLI tool of `chan` only.

* **Is releasing software a pain point for you (if it is, what's the worst part,
  add some comments)?**

  Not at all, we release very often.

* **Do you share your release notes with every release?**

  Our current setup is to publish a global changelog for all apps at once, using
  the minor version as a base. Patch versions are per-app (per git repository).
  The Elixir repository has its own changelog maintained with chan. Project
  changelogs are purely tehcnical while the global changelog focuses on features
  and interop. It is made for the other teams we work with.


* **Are you releasing against self-hosted infra (ie: digital-ocean-alike, custom
  hard/soft) or using cloud services (aws, vercel, netlify)?**

  We have a Kubernetes cluster but I do not know much about it. Basically I
  release to Gitlab CI :D .

* **If interested in enterprise support, would you help us connect with your
  company management?**

  I would but we are not in need for support at the moment, as the CLI tool is
  just what we need.
